source/date_time.md

```diff
@@ -0,0 +1,89 @@
+calendar-style date picker will be shown for selecting the date.
+
+The “Time” question type is for capturing time values, for example on a question
+like “At what time do you leave for work?” In both KoboCollect and Enketo, a
+time picker is shown where a user can select their response.
+
+The third type “Date and time” is for capturing both date and time responses on
+a single question.
+
+## How to set up Date and Time question types
+
+### Setting up in FormBuilder
+
+Adding “Date”, “Time”, and “Date and Time” questions is simple and does not
+require any extra setup:
+
+-   In the KoboToolbox Formbuilder, click the + button to add a new question
+-   Type the question text, for example “What is your date of birth?”, then
+    click ADD QUESTION or press ENTER on your keyboard
+-   Choose the question type
+
+![Adding the questions](images/date_time/adding.gif)
+
+### Setting up in XLSForm
+
+To add date, time, and date and time questions in the XLSForm, use the date,
+time and dateTime question types as shown in the example below:
+
+In XLSForm, you can set up the following:
+
+| type     | name      | label                                         |
+| :------- | :-------- | :-------------------------------------------- |
+| date     | dob       | On what date were you born?                   |
+| time     | time      | At what time do you you leave for work?       |
+| dateTime | date_time | At what date and time did the training start? |
+
+## Appearance of date and time question types in web forms and KoboCollect
+
+### Default appearance
+
+![Default appearances](images/date_time/default_appearances.png)
+
+### Advanced appearances
+
+When adding the “Date” question type, you can choose from a number of display
+options (under the question settings). Appearances change the way the question
+is displayed on web forms and on KoboCollect.
+
+![Default appearances](images/date_time/advanced_appearance.png)
+
+![Default appearances](images/date_time/advanced_appearances.png)
+
+### Adding custom appearances for date questions in XLSForm
+
+You can specify advanced appearances in XLSForm through the appearance column as
+follows:
+
+| type | name         | label                              | appearance |
+| :--- | :----------- | :--------------------------------- | :--------- |
+| date | rains_start  | When did the planting rains start? | month-year |
+| date | year_migrate | In which year did you migrate?     | year       |
+
+## Using date and time questions in custom logic
+
+When defining custom skip logic (relevant), validation criteria (constraint),
+and mandatory response criteria (required) using xlsform code, date values
+should be included using the `date()` function, and in the format
+`“YYYY-MM-DD”`. For example, if you are creating validation criteria on a date
+question so that all survey responses must be before the date “April 10th,
+2022”, your validation logic will be `.<=date('2022-04-11')`.
```

Comment:
The description of the constraint doesn't match the code. Should rather be `. < date('2022-04-10')`

Reply:
Done